### ISSUE TITLE
Are container tab markers supposed to be on top of tabs now?

### ISSUE THREAD

**Issue by PoisonFrog** [mozilla/multi-account-containers#2062]
I don't know if it is supposed to look like this in Firefox 89 or if this is a bug. It doesn't look good on the new design. If this is intentional is there a way to keep the containers but remove the colored dash in tabs? I hate this new look.

This is how it looks like in my Firefox: https://i.imgur.com/GVumnDt.png

**Comment by nanaya** [comment]
On Windows 8 it's now barely visible:

![](https://s.myconan.net/2021-06/firefox_2021-06-03_12-29-57.png)

Compare that to how it looked like before:

![](https://s.myconan.net/2021-06/mstsc_2021-06-03_12-32-06.png)

**Comment by mschultz-tls** [comment]
I agree with comments above.  The pencil thin colored line on the top of the tabs is not enough of an indicator.  It's way to subtle.  Bring back bold tab color we had prior to FF 89.  This has made FF almost impossible to use.

**Comment by AussieFred** [comment]
It's quite hard for the eye to see the new bar and discern it against the edge of the window overall causing added eye strain. If moving the bar to the top of the tab was intentional, please can we change it back?

**Comment by mrjohnson22** [comment]
Tab highlights are controlled by core Firefox itself, not the Multi-Account Containers extension, aren't they?

In any case, the highlight can be moved via tweaks to userChrome.css. [Here are some suggestions.](https://old.reddit.com/r/FirefoxCSS/comments/nvshmx/change_position_of_container_tabs/)

I use the following to also re-enable the active tab highlight on top:
```
/*** Container indicator to bottom of tab ***/
/* Active tab color for container tabs will use container color */
#tabbrowser-tabs .tab-context-line {
    display: none;
}
/* Container line */
#tabbrowser-tabs .tab-background {
    border-bottom: solid 2px var(--identity-icon-color) !important;
}
/* Active tab highlight line */
#tabbrowser-tabs .tab-background[selected="true"]  {
    border-top: solid 2px var(--focus-outline-color) !important;
}
```
![Screenshot from 2021-06-12 11-56-45](https://user-images.githubusercontent.com/8988672/121781992-72a59b00-cb75-11eb-9d97-55723391d461.png)
![Screenshot from 2021-06-12 11-57-04](https://user-images.githubusercontent.com/8988672/121781994-73d6c800-cb75-11eb-8578-234ce23a205b.png)
![Screenshot from 2021-06-12 11-57-27](https://user-images.githubusercontent.com/8988672/121781996-7507f500-cb75-11eb-8194-ebda8c0f50a8.png)

**Comment by PoisonFrog** [comment]
> Tab highlights are controlled by core Firefox itself, not the Multi-Account Containers extension, aren't they?
> 
> In any case, the highlight can be moved via tweaks to userChrome.css. [Here are some suggestions.](https://old.reddit.com/r/FirefoxCSS/comments/nvshmx/change_position_of_container_tabs/)
> 
> I use the following to also re-enable the active tab highlight on top:
> 
> ```
> /*** Container indicator to bottom of tab ***/
> /* Active tab color for container tabs will use container color */
> #tabbrowser-tabs .tab-context-line {
>     display: none;
> }
> /* Container line */
> #tabbrowser-tabs .tab-background {
>     border-bottom: solid 2px var(--identity-icon-color) !important;
> }
> /* Active tab highlight line */
> #tabbrowser-tabs .tab-background[selected="true"]  {
>     border-top: solid 2px var(--focus-outline-color) !important;
> }
> ```
> 
> ![Screenshot from 2021-06-12 11-56-45](https://user-images.githubusercontent.com/8988672/121781992-72a59b00-cb75-11eb-9d97-55723391d461.png)
> ![Screenshot from 2021-06-12 11-57-04](https://user-images.githubusercontent.com/8988672/121781994-73d6c800-cb75-11eb-8578-234ce23a205b.png)
> ![Screenshot from 2021-06-12 11-57-27](https://user-images.githubusercontent.com/8988672/121781996-7507f500-cb75-11eb-8194-ebda8c0f50a8.png)

Thanks! This actually solved the problem. I'm using the userChrome.css someone posted on that reddit thread and it looks like this:
![image](https://user-images.githubusercontent.com/74987652/121788521-6a714e00-cbbd-11eb-8aa1-7dc82698fc5f.png)

I find this very minimal and elegant, if someone want the same:

```
/** Container Tab - Color line at icon's bottom *****************************/
.tab-context-line {
display: none;
}
.tab-icon-image {
box-sizing: content-box;
padding: 3px 0;
border-bottom: 2px solid var(--identity-icon-color);
}
```
[Original author github page](https://github.com/black7375/Firefox-UI-Fix/blob/v1.0/userChrome.css)
